### PR TITLE
Add post changes

### PR DESCRIPTION
**Pull request by muhammadqaisar74**
### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: common/static/common/templates/discussion/templates.underscore [hunk 1]
```diff
@@ -775,24 +838,28 @@
             <%- gettext("Post type") %>
         </span>
     </legend>
-    <div class="field-help" id="field_help_post_type">
+    <!-- <div class="field-help" id="field_help_post_type">
         <%- gettext("Questions raise issues that need answers. Discussions share ideas and start conversations. (Required)") %>
-    </div>
+    </div>-->
```
Comment (m-usman01) [inline common/static/common/templates/discussion/templates.underscore:843]: Will this code be used later or is this an unnecessary comment.

Comment (muhammadqaisar74) [reply common/static/common/templates/discussion/templates.underscore:843]: I think this will be used later

**Review by m-usman01 — APPROVED** [review]

File: common/static/common/templates/discussion/templates.underscore [hunk 2]
```diff
@@ -582,7 +582,7 @@
 <% } %>
 </script>
 <script aria-hidden="true" type="text/template" id="topic-template">
-<div class="field-label">
+<!--<div class="field-label">
```
Comment (emzubair) [inline common/static/common/templates/discussion/templates.underscore:585]: Have you verified the comment works?

File: common/static/common/templates/discussion/templates.underscore [hunk 3]
```diff
@@ -775,24 +838,28 @@
...
+    <!-- <div class="field-help" id="field_help_post_type">
         <%- gettext("Questions raise issues that need answers. Discussions share ideas and start conversations. (Required)") %>
-    </div>
+    </div>-->
     <div class="field-label">
-        <label for="<%- edx.StringUtils.interpolate(form_id) %>-post-type-question" class="post-type-label">
-            <input aria-describedby="field_help_post_type" type="radio" name="<%- edx.StringUtils.interpolate(form_id) %>-post-type" class="field-input input-radio" id="<%- edx.StringUtils.interpolate(form_id) %>-post-type-question" value="question">
-            <span class="field-input-label">
-                <span class="icon fa fa-question" aria-hidden="true"></span>
-                <% // Translators: This is a forum post type %>
```
Comment (emzubair) [inline common/static/common/templates/discussion/templates.underscore:786]: 👆🏽 this is the correct way to comment underscore files

**Review by emzubair — CHANGES_REQUESTED** [review]

**Pull request merged**